src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -16,12 +16,24 @@ import {
   FullHeader,
 } from "v2/Apps/Conversation/Components/InboxHeaders"
 import { DetailsFragmentContainer as Details } from "../../Components/Details"
+import styled from "styled-components"
 interface ConversationRouteProps {
   me: Conversation_me
   conversationID: string
   match: Match
 }
 
+const ConstrainedHeightFlex = styled(Flex)`
+  height: calc(100vh - 145px);
```

Comment:
Just curious what does the `145px` represent?

Reply:
Oh is that accounting for the header thing that contains Inbox, Conversation with..., etc?

Reply:
It is indeed. Might be good to add a comment here to account for that.